egs/icsi/s5/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+# Pawel: this one starts from the best tdnn model for ami, those are fairly similar
+# datasets so I would expect similar modelling tricks will work for both
+
+set -e -o pipefail
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=0
+mic=ihm
+nj=30
+min_seg_len=1.55
```

Comment:
Recent scripts don't need min_seg_len and can instead provide multiple frames-per-chunk like 160,140,100 etc. The AMI recipe is also not updated with this change. But this is something that can be done in this recipe to reduce the many complexities involved in combing short segment feats.